src/bytes.jl

```diff
@@ -53,7 +53,11 @@ for c in 0x00:0xFF
     elseif haskey(REVERSE_ESCAPES, c)
         [BACKSLASH, REVERSE_ESCAPES[c]]
     elseif iscntrl(Char(c)) || !isprint(Char(c))
-        UInt8[BACKSLASH, LATIN_U, hex(c, 4)...]
+        if VERSION < v"0.7.0-DEV.4446"
+            UInt8[BACKSLASH, LATIN_U, hex(c, 4)...]
+        else
+            UInt8[BACKSLASH, LATIN_U, string(c, base=16, pad=4)...]
```

Comment:
can we Compat-ify this method of string? maybe not easily...

Reply:
Yes, looked into that, but I could not work around the stack overflows resulting from overwriting Base.string. I don't know if it'S possible to provide proper Compat versions of this.

I would suggest, as this is the only place where this is used, to merge as-is and revisit later.